MyCoolApi.Tests/MyCoolApi.Tests.csproj

```diff
@@ -1,20 +1,33 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>net9.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
     <EnableMSTestRunner>true</EnableMSTestRunner>
+    <!--
+	    If .NET 10 SDK is installed, this property can be removed and instead you can enable the MTP dotnet test experience,
+		see https://learn.microsoft.com/dotnet/core/tools/dotnet-test?tabs=dotnet-test-with-vstest#description.
+	-->
     <TestingPlatformDotnetTestSupport>true</TestingPlatformDotnetTestSupport>
     <IsPackable>false</IsPackable>
-
+	<!-- 
+	    This cannot be done automatically through activation of `EnableMSTestRunner` because of some limitation in MSBuild.
+		The change is relatively complex as it would involve changes in .NET SDK and NuGet.
+	-->
+	<OutputType>Exe</OutputType>
   </PropertyGroup>
```

Comment:
You may also be interesting in using the following 2 properties:
<TestingPlatformShowTestsFailure>true</TestingPlatformShowTestsFailure>
<TestingPlatformCaptureOutput>false</TestingPlatformCaptureOutput>